src/lectern-client/schema-rest-client.ts

```diff
@@ -19,9 +19,19 @@ export const schemaClient: SchemaServiceRestClient = {
       L.debug(`in fetch schema ${version}`);
       const result = delay(1000);
       const dictionary = await result(() => {
-        const dictionaryJson = require(schemaSvcUrl.substring(7, schemaSvcUrl.length));
-        return dictionaryJson;
+        const dictionaries: SchemasDictionary[] = require(schemaSvcUrl.substring(
+          7,
+          schemaSvcUrl.length,
+        )) as SchemasDictionary[];
+        const dic = dictionaries.find((d: any) => d.version == version);
```

Comment:
and I guess we assume that all the schema in stub schema array have same `name`?? (might help to note that too)

Reply:
the name was never considered for stub schema not for any specific reason, like it fetches anything

Reply:
updated the file loader to consider the name